include/zephyr/drivers/firmware/scmi/protocol.h

```diff
@@ -17,6 +17,25 @@
 #include <stdint.h>
 #include <errno.h>
 
+#define SCMI_MAX_STR_SIZE        64
+#define SCMI_SHORT_NAME_MAX_SIZE 16
+
```

Comment:
I think should be in a separate patch. Initial patch should be for removing SCMI_FIELD_MAKE.

Reply:
I've moved defines in Base proto patch (first user) and retested build of every patch.